core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProvider.scala

```diff
@@ -88,13 +88,16 @@ class LocalEntityProvider(workspace: Workspace, implicit protected val dataSourc
   override def deleteEntities(entRefs: Seq[AttributeEntityReference]): Future[Int] = {
     dataSource.inTransaction { dataAccess =>
       // withAllEntityRefs throws exception if some entities not found; passes through if all ok
-      withAllEntityRefs(workspaceContext, dataAccess, entRefs) { _ =>
-        dataAccess.entityQuery.getAllReferringEntities(workspaceContext, entRefs.toSet) flatMap { referringEntities =>
-          if (referringEntities != entRefs.toSet)
-            throw new DeleteEntitiesConflictException(referringEntities)
-          else {
-            dataAccess.entityQuery.hide(workspaceContext, entRefs)
-          }
+      traceDBIO("LocalEntityProvider.deleteEntities") { rootSpan =>
+        rootSpan.putAttribute("workspaceId", OpenCensusAttributeValue.stringAttributeValue(workspaceContext.workspaceId))
```

Comment:
add the number of entities:
```suggestion
        rootSpan.putAttribute("workspaceId", OpenCensusAttributeValue.stringAttributeValue(workspaceContext.workspaceId))
        rootSpan.putAttribute("numEntities", OpenCensusAttributeValue.longAttributeValue(entRefs.length))
```